doc/source/user_guide/machine_learning_pysteps.rst

```diff
@@ -0,0 +1,103 @@
+.. _machine_learning_pysteps:
+
+Machine learning vs pysteps
+===========================
+How to correctly compare the accuracy of machine learning against pysteps?
+
+Before starting, you need to ask yourself these two questions:
```

Comment:
Starting what? Is this referring to the comparison with extrapolation or to the actual training of the ML model?